pkgs/top-level/all-packages.nix

```diff
@@ -20817,4 +20817,6 @@ with pkgs;
   safeDiscardStringContext = callPackage ../build-support/safe-discard-string-context.nix { };
 
   simplehttp2server = callPackage ../servers/simplehttp2server { };
+
+  diceware = pythonPackages.callPackage ../tools/security/diceware { };
```

Comment:
And here `pythonPackages` isn't needed in that case. Sorry for the mess @asymmetric.

Reply:
No worries :)